src/fusion_power_plant.h

```diff
@@ -106,6 +106,16 @@ class FusionPowerPlant : public cyclus::Facility  {
   }
   double sequestered_equilibrium; 
 
+  #pragma cyclus var { \
+    "default": 0.9, \
+    "doc": "Fraction of desired startup tritium inventory required. ", \
+    "tooltip": "Gives flexibility accounting for decay", \
+    "units": "Dimensionless", \
```

Comment:
Not sure if this matters?
```suggestion
    "units": "dimensionless", \
```